src/Models/NestedsetCollection.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace A17\Twill\Models;
+
+use Kalnoy\Nestedset\Collection as BaseNestedsetCollection;
+
+class NestedsetCollection extends BaseNestedsetCollection
+{
+    /**
+     * Build a tree from a list of nodes. Each item will have set children relation.
+     *
+     * To successfully build tree "id", "_lft" and "parent_id" keys must present.
+     *
+     * If `$root` is provided, the tree will contain only descendants of that node.
+     *
+     * @param mixed $root
+     *
+     * @return Collection
+     */
+    public function toTree($root = false)
+    {
+        if ($this->isEmpty()) {
+            return new static();
+        }
+
+        $this->linkNodes();
+
+        $items = [];
+
+        $root = $this->getRootNodeId($root);
+
+        $ids = collect($this->items)->pluck('id')->toArray();
+
+        /** @var Model|NodeTrait $node */
```

Comment:
NodeTrait is not imported so editors will not understand the typecast. It should be either imported or fully namespaced.

Reply:
@haringsrob  I have imported it